src/Libraries/GeometryUI/PanelSurfaceBoundaryConditions.cs

```diff
@@ -0,0 +1,79 @@
+using Autodesk.DesignScript.Geometry;
+using Autodesk.DesignScript.Runtime;
+using CoreNodeModels;
+using DSCore;
+using Dynamo.Graph.Nodes;
+using Dynamo.Utilities;
+using Newtonsoft.Json;
+using ProtoCore.AST.AssociativeAST;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace GeometryUI
+{
+    [IsVisibleInDynamoLibrary(true)]
+    [NodeName("PanelSurfaceBoundaryCondition")]
+    [NodeCategory("Geometry.PanelSurface")]
+    [NodeDescription("PanelSurfaceBoundaryConditionDropDownDesc", typeof(Properties.Resources))]
+    [OutPortNames(">")]
+    [OutPortTypes(nameof(PanelSurfaceBoundaryCondition))]
+    [OutPortDescriptions("PanelSurface BoundaryCondition enum value")]
+    [IsDesignScriptCompatible]
+    public class PanelSurfaceBoundaryConditionDropDown : DSDropDownBase
+    {
+        public PanelSurfaceBoundaryConditionDropDown(): base(">") {
+            SelectedIndex = (int)PanelSurfaceBoundaryCondition.Keep;
+        }
+
+        [JsonConstructor]
+        protected PanelSurfaceBoundaryConditionDropDown(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base(">", inPorts, outPorts) { }
+
+        /// <summary>
+        /// Overrides the default behavior to serialize internal enumeration id 
+        /// instead of the name of the enum constant.
+        /// </summary>
+        /// <param name="item">Selected DynamoDropDownItem</param>
+        /// <returns>A string representing the internal enum id.</returns>
+        protected override string GetSelectedStringFromItem(DynamoDropDownItem item)
+        {
+            return item == null ? string.Empty : item.Item.ToString();
+        }
+
+        protected override SelectionState PopulateItemsCore(string currentSelection)
+        {
+            Items.Clear();
+            foreach (var constant in Enum.GetValues(typeof(PanelSurfaceBoundaryCondition)))
+            {
+                Items.Add(new DynamoDropDownItem(constant.ToString(), constant));
+            }
+
+            Items = Items.OrderBy(x => x.Name).ToObservableCollection();
```

Comment:
Are these being ordered by name because that order just happens to match this order?
```
Keep
Remove
RemoveVertices
```

Reply:
Yeah, the order of the enum values just happen to match the order you posted above